src/services/workshops/hooks.js

```diff
@@ -19,8 +19,13 @@ async function assignToOwner({ params: { user: owner }, app, result, service })
 async function checkXMLChanged(context) {
     const workshop = await context.app.service('workshops').get(context.id);
 
-    if (context.data.hasOwnProperty('xml')) {
-        workshop.xml !== context.data.xml ?
+    if (context.data.hasOwnProperty('xml') || context.data.hasOwnProperty('properties')) {
+        const hasWorkshopChanged = (
+            workshop.xml !== context.data.xml
+            || workshop.properties !== context.data.properties
```

Comment:
Note that this will always be `true` when `workshop.properties` and `context.data.properties` are not the same reference, thus the signal will be sent even if the `properties` is unchanged.